experimental/dds/tree2/src/test/shared-tree/sharedTree.bench.ts

```diff
@@ -388,6 +388,27 @@ describe("SharedTree benchmarks", () => {
 			});
 		}
 	});
+
+	describe("acking local commits", () => {
+		const localCommitSize = [1, 25, 100, 500, 1000];
+		for (const size of localCommitSize) {
+			benchmark({
+				type: BenchmarkType.Measurement,
+				title: `for ${size} local commit${size === 1 ? "" : "s"}`,
+				benchmarkFn: () => {
+					const provider = new TestTreeProviderLite();
+					const [tree] = provider.trees;
+					for (let i = 0; i < size; i++) {
+						insert(tree, i, i);
+					}
+
+					for (let i = 0; i < size; i++) {
+						provider.processMessages(1);
```

Comment:
Given the title of this test, I would expect us to only be measuring this loop here. Can we create the tree and do all the inserts in the benchmark `before` block? Also, you can remove the loop here and just do `provider.processMessages()`

Reply:
Unfortunately, we cannot. This was actually causing the error I saw before, the before block only gets run once. This is probably a good argument for adding a beforeEach to benchmark though.

Reply:
Oh darn... I see... hmm. It's just not great that we are also measuring the time taken to create the mock runtime and do all the inserts. It seems like that'd enough work to skew the measurement. Have you asked Craig or Taylor or someone if there's a way to do something closer to what we want? It seems like we would have run into this before. I'm fine with this being checked in as-is as long as we follow up and either make sure it can't be done better, or improve it later.

Reply:
Sounds good. I'll add a todo and ask Craig.

Reply:
I just noticed `benchmarkFnCustom` that is used by other tests in this file. It looks like it can do what we want.